components/postmark/actions/send-single-email/send-single-email.mjs

```diff
@@ -1,92 +1,50 @@
 // legacy_hash_id: a_8KiGrJ
-import { axios } from "@pipedream/platform";
+import postmark from "../../postmark.app.mjs";
+
+let objSharedProps = {};
+postmark.methods.listSharedProps().forEach((propName) => {
+  objSharedProps[propName] = {
+    propDefinition: [
+      postmark,
+      propName,
+    ],
+  };
+});
```

Comment:
Good thinking in reusing common props!
There is a pattern [described in the docs](https://pipedream.com/docs/components/guidelines/#common-files-optional) that we use to achieve this though. You can take a look at [Google Drive](https://github.com/PipedreamHQ/pipedream/tree/master/components/google_drive) and [AWS](https://github.com/PipedreamHQ/pipedream/tree/master/components/aws) components as examples

Reply:
As seen in this [comment](https://github.com/PipedreamHQ/pipedream/pull/2823/files#r873974530)